ObjCVoiceCallKitQuickstart/ViewController.m

```diff
@@ -428,29 +428,16 @@ - (void)performVoiceCallWithUUID:(NSUUID *)uuid
     
     self.call = [TwilioVoice call:[self fetchAccessToken]
                            params:@{}
+                             uuid:uuid
                          delegate:self];
-    
-    if (!self.call) {
-        completionHandler(NO);
-    } else {
-        self.call.uuid = uuid;
-    }
-    
     self.callKitCompletionCallback = completionHandler;
 }
 
 - (void)performAnswerVoiceCallWithUUID:(NSUUID *)uuid
                             completion:(void(^)(BOOL success))completionHandler {
```

Comment:
Is the UUID argument needed? Effectively this app supports a single `self.callInvite` at a time. Maybe we should just assert that the UUID from CallKit is the same as ours?

`NSAssert([self.callIInvite.uuid isEqual:uuid], @"We only support one Invite at a time.");`